unionvisor/src/testdata/test_create_and_read/bins/bin.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env sh
 set -e
 
-mkdir -p $4
-printf %s '{"name": "upgrade1", "height": 123}' > $4/upgrade-info.json
```

Comment:
https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing

```sh
shellcheck error:

In unionvisor/src/testdata/test_create_and_read/bins/bin.sh line 4:
mkdir -p $4
         ^-- SC2086 (info): Double quote to prevent globbing and word splitting.
         
printf %s '{"name": "upgrade1", "height": 123}' > $4/upgrade-info.json
                                                  ^-- SC2086 (info): Double quote to prevent globbing and word splitting.
```